src/frontend/src/repositories/anchorMetadata.ts

```diff
@@ -0,0 +1,168 @@
+import { MetadataMapV2 } from "$generated/internet_identity_types";
+import type { AuthenticatedConnection } from "$src/utils/iiConnection";
+
+export type AnchorMetadata = {
+  recoveryPageShownTimestampMillis?: number;
```

Comment:
Doesn't this need to be `bigint`?

Reply:
I will get it from the browser with `Date.now()` which returns a `number`.

From chatGPT:

> Date.now() will return Number.MAX_SAFE_INTEGER sometime around the year 287,586

If we were using nanoseconds, then yes. But with milliseconds it should be enough.